src/assets/i18n/de.json

```diff
@@ -0,0 +1,237 @@
+  },
+  "LOGOUT_PROFILE_TEXT": "Ausloggen?",
+  "MARKETS_PAGE": {
+    "MARKET_CAP": "Markt Cap.",
+    "MARKET_DATA": "Markt Daten",
+    "PERCENTAGE_CHANGE": "24h Änderung",
+    "PRICE": "Preis",
+    "VOLUME": "Volumen"
+  },
+  "NAME": "Name",
+  "NETWORKS_PAGE": {
+    "CHANGE_PEER": "Ändere Peer",
+    "CHOOSE_ONE": "Wähle einen der Netzwerk Typen",
+    "DELAY": "Verzögerung",
+    "HEIGHT": "Höhe",
+    "LOOKING_GOOD_PEER": "Suche nach einem guten Peer",
+    "PEER_SUCCESSFULLY_CHANGED": "Peer erfolgreich geändert",
+    "NETWORK_STATUS": "Netzwerk Status",
+    "NETWORK_TYPE": "Netzwerk Typ",
+    "INTERNET_DESCONNECTED": "Internet Verbindung nicht möglich!",
+    "PEER": "Peer"
+  },
+  "NEXT": "Nächste",
+  "NO": "Nein",
+  "PIN_CODE": {
+    "CONFIRM": "Bestätige deinen PIN",
+    "CREATE": "Erstelle deinen PIN",
+    "DEFAULT_MESSAGE": "Gib deinen PIN ein",
+    "LOGIN_ERROR": "Es gab ein Problem beim Einloggen mit deinem PIN",
+    "PIN_CREATED_ERROR_TEXT": "Es gab ein Problem beim Erstellen deines PINs",
+    "PIN_CREATED_TEXT": "Dein PIN wurde erstellt",
+    "PIN_UPDATED_ERROR_TEXT": "Es gab ein Problem beim Aktualisieren deines PINs",
+    "PIN_UPDATED_TEXT": "Dein PIN wurde aktualisiert",
+    "TYPE_PIN_ENCRYPT_PASSPHRASE": "Gib deinen PIN ein, um deine Passphrase zu aktualisieren",
+    "TYPE_PIN_SIGN_TRANSACTION": "Gib deinen PIN ein, um deine Transaktion zu bestätigen",
+    "WRONG": "Gib bitte deinen korrekten PIN ein"
+  },
+  "PROFILES_PAGE": {
+    "ADD_PROFILE_ERROR": "Konnte Porfil nicht hinzufügen",
+    "ADD_PROFILE_TEXT": "Klick den nachfolgenen Button, um ein Profil zu erstellen",
+    "CREATE_PROFILE": "Erstelle Profil",
+    "DELETE_NOT_EMPTY": "Du kannst kein Profil löschen, dass 1 oder mehrere Wallets besitzt",
+    "GET_STARTED": "Beginne",
+    "SIGN_IN": "Einloggen"
+  },
+  "QR_CODE": {
+    "INVALID_QR_ERROR": "Es gab ein Problem beim Scannen eines QR Codes",
+    "INVALID_QRCODE": "Dieser QR Code hat keine gültige Adresse oder Passphrase",
+    "PERMISSION_DENIED": "QR Kamera abgelehnt",
+    "PERMISSION_PERMANENTLY_DENIED": "QR Kamera zurzeit abgelehnt. Gehe zu den Geräte Einstellungen, um dies zu ändern.",
+    "PROBLEM_TEXT": "Es gab ein Problem beim Scannen eines QR Codes"
+  },
+  "RENAME": "Umbennenen",
+  "SAVE": "Speicher",
+  "SEARCH": "Suchen",
+  "SETTINGS_PAGE": {
+    "ABOUT": "Über",
+    "APPEARANCE": "Erscheinung",
+    "APPLICATION": "Applikation",
+    "CHANGE_PIN": "Ändere PIN",
+    "CLEAR_DATA": "Lösche Daten",
+    "CLEAR_DATA_TEXT": "Alle deine Daten, inklusive Profile, Wallets und Kontakte werden von der App gelöscht und zurückgesetzt.",
+    "CURRENCY": "Währung",
+    "DARK_MODE": "Dark Mode",
+    "LANGUAGE": "Sprache",
+    "PRIVACY_POLICY": "Privatsphäre",
+    "SECURITY": "Sicherheit",
+    "SELECT_WALLET_FIRST_TEXT": "Wähle zuerst ein Wallet auf der Wallets Seite",
+    "NOT_AVAILABLE_WATCH_ONLY": "Nicht verfügbar für reine Leseadresse",
```

Comment:
`Leseadresse` = `Watch-Only Adresse`

Reply:
ah ok -> was totaly unsure if I should translate 'Watch-Only' to german or not